plugins/experimental/uri_signing/README.md

```diff
@@ -130,7 +130,7 @@ The following claims are understood:
   - `cdniuc`: Validated last, after key verificationD. **Only `regex` is supported!**
   - `cdniets`: If cdnistt is 1, this must be present and non-zero.
   - `cdnistt`: If present, must be 1.
-  - `cdnistd`: If present, must be 0.
+  - `cdnistd`: Renwal token cookies will have cdnistd path segments of the request in their path attribute. 
```

Comment:
Renewal (and trailing whitespace here and other places where the clang-format ci test failed)

Reply:
Should be resolved

Reply:
[approve ci]